pkg/webui/console/views/device-overview/index.js

```diff
@@ -47,18 +53,26 @@ const m = defineMessages({
   keysNotExposed: 'Keys are not exposed',
   failedAccessOtherHostDevice:
     'The end device you attempted to visit is registered on a different cluster and needs to be accessed using its host Console.',
+  macData: 'Download MAC data',
+  hasSession:
+    'The MAC data can contain sensitive information such as session keys that can be used to decrypt messages. Do not share this information publicly.',
```

Comment:
```suggestion
    'The MAC data can contain sensitive information such as session keys that can be used to decrypt messages. <b>Do not share this information publicly</b>.',
```
Let's stress on that a bit more.